src/Header.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Header() {
+  return (
+    <header>
+      <Link>
+        <img src="./logo.svg" alt="Logo" />
+      </Link>
+      <ul>
```

Comment:
You could wrap this into a `nav` element I guess.

Reply:
and done. thanks